### DISCUSSION TITLE
Theme isnt taking effect in production

### DISCUSSION THREAD

**Discussion by mohamedallam1991** [jpanther/congo#286] [Q&A]
Howdy,

First I thank you guys for this awesome project, I really love this theme, more than a bunch of other Hugo themes out there specially how much you can customize.

My question please, is that: even in the  `config/_default/params.toml` everything is set properly, the theme Ocean, or the exact same theme I reproduced under `Sky.css`, It's not taking effect in production.

[Params file repo](https://github.com/mohamedallam1991/GoBook/blob/master/config/_default/params.toml)

Deployment:
[Github Pages](https://mohamedallam1991.github.io/GoBook/)
[Cloudflare Pages](https://gobook.pages.dev)

Whats the problem please? Thank you in advance

**Comment by rshmhrj** [comment]
Hi @mohamedallam1991,
https://github.com/mohamedallam1991/GoBook/blob/master/config/_default/params.toml#L8 shows the theme set to `ocean`.  Can you try changing it to `sky`?

E.g.:
```toml
colorScheme = "sky"
```

**Reply by mohamedallam1991** [reply]
Hello Rishi,
Sorry for the late reply, so now it took effect, for whatever reason, so no problems there thank you.

I want to point out:
- The documentation dosent include the purple favicon used and how to replace is, like in the JPanther blog its JP.
I figured we need to add the icons to the static folder, but with some naming convention.
- There is no background image [socialsharepreview.com](https://socialsharepreview.com/) will show that.
Should be added like this `thumbnail: "img/placeholder.png" # Thumbnail image`
For reference this  theme does support it [mainiroad](https://github.com/Vimux/Mainroad)

Probably Hugo issue:
Today, at glance, I figured even though I deployed to multiple platforms like Cloudflare pages, the robots file, is made with a like to my github.
```xml
User-agent: *
Allow: /
Sitemap: https://mohamedallam1991/github.io/gobook/sitemap.xml
```
**It should be** 
```xml
Sitemap: https://gobook/pages.dev/sitemap.xml
```
Cloudflare command I use:
```
hugo --gc --minify -D -b $CF_PAGES_URL
```

**Reply by rshmhrj** [reply]
Hi @mohamedallam1991 

For the favicon, I think I used an online generator the spit out multiple files and I threw them all in the `static` folder.  When I'm running the site locally, they all show up in the `public` folder.
![image](https://user-images.githubusercontent.com/750874/188835833-76deb3a9-d0ec-4d2c-8a6a-fb5ad80d6bfc.png)

For the background pictures, I used to have it in the frontmatter as `thumbnail` but changed it to `preview`.  I have been trying out a VSCode extension called [frontmatter](https://frontmatter.codes/) which is like a headless CMS inside the IDE, and only after changing to `preview` did the thumbnails display correctly.  Maybe try that and see if it works?
![image](https://user-images.githubusercontent.com/750874/188837800-87c7f87a-0c59-44fa-b2cd-9e3297b27b79.png)

Not sure about the Cloudflare part.  I've been using Gitlab Pages for the CI/CD and hosting.  Maybe someone else would have a suggestion?

**Reply by jpanther** [reply]
> The documentation dosent include the purple favicon used and how to replace is, like in the JPanther blog its JP.

There's literally a section in the theme documentation on Favicons that explains how to do it. This is the exact process I used to generate my "JP" icons. Please refer to the [favicons documentation](https://jpanther.github.io/congo/docs/partials/#favicons).

> There is no background image [socialsharepreview.com](https://socialsharepreview.com/) will show that.

The theme uses the default Hugo social sharing templates and so the standard Hugo front matter parameters are supported. If you provide paths to the images you wish to use in an `images` parameter you should find they are displayed when sharing to social networks. Both open graph and Twitter cards are supported.

```yaml
images: ["/path/to/image1", "/path/to/image2"]
```

**Reply by erwinlauener** [reply]
Thanks for the nice theme. Everything is good, the command 'Hugo server' shows 'ocean' structure and colors correctly (localhost). Unfortunately index.html  doesn't work in production/server (public files). CSS isn't working. Any suggestions?
